src/PowerShellEditorServices/Session/RunspaceDetails.cs

```diff
@@ -225,8 +225,11 @@ internal bool HasCapability<TCapability>()
                             defaultValue: string.Empty);
 
                 // hostname is 'ServerRemoteHost' when the user enters a session.
-                // ex. Enter-PSSession, Enter-PSHostProcess
-                if (hostName.Equals("ServerRemoteHost", StringComparison.Ordinal))
+                // ex. Enter-PSSession
+                // Attaching to process currently needs to be marked as a local session
+                // so we skip this if block if the runspace is from Enter-PSHostProcess
+                if (hostName.Equals("ServerRemoteHost", StringComparison.Ordinal)
+                    && runspace.OriginalConnectionInfo.GetType().ToString() != "System.Management.Automation.Runspaces.NamedPipeConnectionInfo")
```

Comment:
May be a good idea to do a null check here.  I think there are a couple `ConnectionInfo`'s that do not get written here but are still `ServerRemoteHost`

```suggestion
                    && runspace.OriginalConnectionInfo != null
                    && runspace.OriginalConnectionInfo.GetType().ToString() != "System.Management.Automation.Runspaces.NamedPipeConnectionInfo")
```

Reply:
better yet:

```
runspace.OriginalConnectionInfo?.GetType().ToString() != "System.Management.Automation.Runspaces.NamedPipeConnectionInfo"
```